theseus/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2018"
 [dependencies]
 thiserror = "1.0"
 async-trait = "0.1.51"
+records = "0.2.0"
```

Comment:
Your library is cool, but I don't think it's a good fit for the project (mainly unneeded and when I want to develop it it's not auto completed well. I think sticking to vanilla structs is fine.

Reply:
Alright, will remove.